### ISSUE TITLE
UnsupportedClassVersionError on JDK 8

### ISSUE THREAD

**Issue by ngbinh** [irundaia/sbt-sassify#29]
when I tried version `1.4.10` on JDK 8, I encountered this error:

```
error 11:55:59:AM] trace: throwable: java.lang.UnsupportedClassVersionError: org/irundaia/sass/jna/SassLibrary has been compiled by a more recent version of the Java Runtime (class file version 53.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

Looks like the library was published with JDK 9?

**Comment by LogicalTime** [comment]
Yep, looks like it was fixed in 1.4.11 though.
https://github.com/irundaia/sbt-sassify/issues/28

**Comment by ngbinh** [comment]
thanks. Indeed!
